serverless-plugin/util.js

```diff
@@ -66,6 +66,44 @@ function resolveEcsClusterNameForSub (cluster) {
   return cluster
 }
 
+/**
+ * Given CloudFormation syntax for an Target Group, derive CloudFormation syntax for
+ * the LoadBalancer LogicalId name
+ *
+ * @param  loadBalancerResources  syntax for an Load Balancer Application
+ * @returns CloudFormation syntax for the Load Balancer Resources
+ */
+function resolveLoadBalancerLogicalIdName (loadBalancerResources) {
```

Comment:
This function is unlikely to help us with what we want to achieve for this feature. We do not want to find the first load balancer. We must find the _right_ load balancer in every case.

Reply:
I see. That is right. Try to find the logic for this implementation with the help of the [diagram](https://github.com/fourTheorem/slic-watch/issues/89).